types/micromodal/micromodal-tests.ts

```diff
@@ -1,8 +1,8 @@
 import MicroModal, { MicroModalConfig } from 'micromodal';
 
 const config: MicroModalConfig = {
-    onShow: (modal) => { console.log(modal!.id); },
-    onClose: (modal) => { console.log(modal!.id); },
+    onShow: (modal, trigger, event) => { console.log(modal!.id);console.log(trigger);console.log(event); },
+    onClose: (modal, trigger, event) => { console.log(modal!.id);console.log(trigger);console.log(event); },
```

Comment:
Spreading these functions across multiple lines should fix the CI errors and make the code a bit more readable:

```suggestion
    onShow: (modal, trigger, event) => {
        console.log(modal!.id);
        console.log(trigger);
        console.log(event);
    },
    onClose: (modal, trigger, event) => {
        console.log(modal!.id);
        console.log(trigger);
        console.log(event);
    },
```